strawberry_django_jwt/utils.py

```diff
@@ -243,3 +244,14 @@ def get_context(
             return ctx.request
         return ctx
     return info
+
+
+def is_event_loop_running():
```

Comment:
This may not be necessary as strawberry-graphql-django already contains [the same function](https://github.com/strawberry-graphql/strawberry-graphql-django/blob/main/strawberry_django/utils.py#L16)

Reply:
Good catch.